src/Controllers/Stats/fullTracklist.php

```diff
@@ -46,9 +46,9 @@
                 'Stats',
                 'fullTracklist',
                 [
-                    'rangesel-starttime' => array_key_exists($_GET['rangesel-starttime']) ?
+                    'rangesel-starttime' => isset($_GET['rangesel-starttime']) ?
                         $_GET['rangesel-starttime'] : '',
-                    'rangesel-endtime' => array_key_exists($_GET['rangesel-endtime']) ? $_GET['rangesel-endtime'] : '',
+                    'rangesel-endtime' => isset($_GET['rangesel-endtime']) ? $_GET['rangesel-endtime'] : '',
```

Comment:
there's quite a lot of this type of thing scattered around the codebase, but perhaps you could use some null coalescing operators here & above? http://stackoverflow.com/a/18603279/995325